### DISCUSSION TITLE
[讨论] 原书重复条目的处理

### DISCUSSION THREAD

**Discussion by ztl8702** [MindongLab/yngdieng#206] [冇讲 (杂谈)]
例如 

https://yngdieng-staging.mindong.asia/feng/p75_16

和

https://yngdieng-staging.mindong.asia/feng/p398_1

是同一个词，解释大体一致，但还是有差别（例句有别）。

**Comment by ztl8702** [comment]
Ideas:

1. 删除其中一个的解释内容，但在feng.txt中保留汉字和读音，内容留空或改成“见 <另一个条目>”。其例句合并进另一个条目。
    1. （将来有可能）“见  <另一个条目>"可做成超链接
2. 两者都保留，因两者对应同一个 zingzeu ID（不久的将来）显示在同一个页面内。即“乍”这个条目内会有三块：1)冯书 p75 的内容 2）冯书 p398 的内容 3）李书的内容
3. 在feng.txt直接删除其中一个（或将其中一个整行注释掉）

**Comment by Guanchishan** [comment]
考虑到榕典目标是成为多来源的词典，那保留原数据结构可能比较合理。支持第二项：

> 2.两者都保留，因两者对应同一个 zingzeu ID（不久的将来）显示在同一个页面内。即“乍”这个条目内会有三块：1)冯书 p75 的内容 2）冯书 p398 的内容 3）李书的内容
